govcd/adminvdc_nsxt_test.go

```diff
@@ -0,0 +1,140 @@
+// +build org functional nsxt ALL
```

Comment:
remove or uncomment ?

Reply:
These are build tags for test. They have to be commented by convention.